### PR TITLE
fix(www): fixes the padding issues on homepage for mobile screen sizes

### PR DESCRIPTION
**Pull request by abhishekjakhar**
### Description

1. The diagram section of homepage is having extra left and right padding(1.5rem) which is making it very narrow on mobile devices, also inconsistent with the other sections on homepage.
2. The features sections of homepage is also narrow because the cards inside of it are having extra left and right padding(1.5rem).
3. The features section is not having any padding bottom on mobile devices which is making it attached to the divider.

The styles have only been touched for devices with size smaller than **mediaQueries.sm**, others have been untouched.

##### Actual
homepage-actual

##### Expected
homepage-expected

##### Actual
homepage-features-actual-1

##### Expected
homepage-features-expected-1

##### Actual
homepage-features-actual-2

##### Expected
homepage-features-expected-2

### How to test

1. Go to the homepage on mobile device
2. Scroll down to the diagram and features section
3. The section should not include the extra padding and should not shrink
4. The features sections should have padding-bottom similar to the padding-top

### CODE REVIEW

File: www/src/components/homepage/homepage-features.js [hunk 1]
```diff
@@ -15,6 +16,10 @@ const HomepageFeatures = () => (
       flex: `0 1 auto`,
       flexWrap: `wrap`,
       px: 8,
+      pb: 8,
```
Comment (fk) [inline www/src/components/homepage/homepage-features.js:19]: Any reason to not use theme-ui's array syntax for `pb` here?

Comment (abhishekjakhar) [reply www/src/components/homepage/homepage-features.js:19]: This is my first PR on Gatsby, so basically I am not aware of that syntax, if you feel doing it with the array syntax is much cleaner way, then I can read about it and update the code, please let me know what you feel?

Comment (abhishekjakhar) [reply www/src/components/homepage/homepage-features.js]: I have used the theme-ui's array syntax for padding-bottom

Comment (fk) [reply www/src/components/homepage/homepage-features.js]: Hey Abhishek! AFK currently, gotta check, but the first position in the `theme-ui` array syntax should result in a "mobile-first" CSS rule without any `media` … hope that makes sense.

Comment (fk) [reply www/src/components/homepage/homepage-features.js]: So the `pb: [8, 0]` should produce something like

```
.element { padding-bottom: 2.5rem } // that's the 8

@media screen and (min-width: 400px)
.element {
    padding-bottom: 0; // that's the 0
}

Comment (abhishekjakhar) [reply www/src/components/homepage/homepage-features.js]: @fk I have exactly same thing which you described above, everything is working perfect.

The 400px and 550px thing which I described above is not related to this PR.

Comment (fk) [reply www/src/components/homepage/homepage-features.js]: Ahh sweet! So let's get this in! <3

**Pull request merged**